Resources/Catalog/FileCatalog.py

```diff
@@ -46,8 +46,10 @@ def __init__( self, catalogs = [], vo = None ):
     self.vo = vo if vo else getVOfromProxyGroup().get( 'Value', None )
 
     self.opHelper = Operations( vo = self.vo )
-    if type( catalogs ) in types.StringTypes:
-      catalogs = [catalogs]
+    if catalogs == None:
```

Comment:
Stephane means that the right way to check if someting is None is "something is None" :). "None" is an object in Python. So this line should read:
if catalogs is None:
...